### PR TITLE
Ravager Balance Tweaks

### PR DESCRIPTION
**Pull request by Surrealaser**
:cl: by Surrealistik
tweak: Rage gained per slash increased to 7.5 up from 5. and Rage gained from damage is now 25% of damage taken down from 30%. Subject to tweaking.
tweak: Ravager base movement speed increased to -0.5 up from -0.4. Progression remains the same.
tweak: Ravager Ravage bonus damage retooled to +25% + 3% per point of rage.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/attack_alien.dm [hunk 1]
```diff
@@ -263,7 +263,7 @@
 	return FALSE
 
 /mob/living/carbon/Xenomorph/Ravager/process_rage_attack()
-	rage += 5 //Gain 5 rage stacks for the attack.
+	rage += 10 //Gain 10 rage stacks for the attack.
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/attack_alien.dm]: move this value into either a config value, define or caste var and put the comments there.

File: code/modules/mob/living/carbon/xenomorph/damage_procs.dm [hunk 2]
```diff
@@ -150,7 +150,7 @@
 /mob/living/carbon/Xenomorph/Ravager/process_rage_damage(damage)
 	if(damage < 1 || world.time < last_damage)
 		return damage
-	rage += round(damage * 0.3) //Gain Rage stacks equal to 30% of damage received.
+	rage += round(damage * 0.25) //Gain Rage stacks equal to 25% of damage received.
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/damage_procs.dm]: move this value into either a config value, define or caste var and put the comments there.

File: code/modules/mob/living/carbon/xenomorph/Powers.dm [hunk 3]
```diff
@@ -1953,7 +1953,7 @@
 		if(target_facing != dir && target_facing != turn(dir,45) && target_facing != turn(dir,-45) ) //Have to be actually facing the target
 			continue
 		if(H.stat != DEAD && !(istype(H.buckled, /obj/structure/bed/nest) && H.status_flags & XENO_HOST) ) //No bully
-			var/extra_dam = rand(xeno_caste.melee_damage_lower, xeno_caste.melee_damage_upper) * (1 + round(rage * 0.01) ) //+1% bonus damage per point of Rage.relative to base melee damage.
+			var/extra_dam = rand(xeno_caste.melee_damage_lower, xeno_caste.melee_damage_upper) * round(0.25 + rage * 0.03, 0.01) //+25% +3% bonus damage per point of Rage.relative to base melee damage.
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/Powers.dm]: move this value into either a config value, define or caste var and put the comments there.

**Comment by AdamElTablawy** [comment]
yeah playing as a rav with this it felt _way_ too good

regular rage took some management to get going-you took damage and maintained rage to go for a risk-reward kind of thing where you weren't at full health when you were angry, but took less damage and did more accordingly

with this, that's not quite the case-being hurt doesn't help ravagers as much, making them squishier, but the moment they manage to start slashing someone they snowball horrifically into a mach-10 engine of death where shooting them barely fazes them and they drop you almost instantly.

this makes ravs _exceptionally_ effective during a push-if a marine falls for some reason all the rav has to do is slash them four times and that + the charge will put them at 50 rage, and they are then perfectly capable of running to the rest of the marines before the rage can wear off and rip into them horribly

**Pull request merged**